src/sa_web/static/js/views/map-view.js

```diff
@@ -40,12 +43,25 @@ var Shareabouts = Shareabouts || {};
         } else if (config.type && config.type === 'landmark') {
           collectionId = config.id;
           self.layers[collectionId] = L.layerGroup();
-          self.landmarkLayerViews[collectionId] = {};
+
+          if (config.visibleDefault) {
+            self.map.addLayer(self.layers[collectionId]);
+          }
```

Comment:
This `config.visibleDefault` block should no longer be necessary, as part of Tanner's new sidebar config. Here is the commit where it has been removed: https://github.com/smartercleanup/platform/commit/04d84b1c63734e209b47165d316b85a71c05b6a4